src/packages/galaxy/galaxy.py

```diff
@@ -271,7 +281,7 @@ def __init__(self, url: str = None):
         if nearest['data']:
             return [neighbor['name'] for neighbor in nearest['data']]
 
-    async def _call(self, endpoint: str, params: Dict[str, str]) -> object:
+    async def _call(self, endpoint: str, params: Dict[str, str] = None) -> object:
```

Comment:
[error] looks like this returns a dictionary, not `object`
```suggestion
    async def _call(self, endpoint: str, params: Dict[str, str] = None) -> Dict:
```

Reply:
b4713e1